eclipse/src/saros/project/FileActivityConsumer.java

```diff
@@ -125,7 +134,8 @@ private void handleFileRecovery(FileActivity activity) throws CoreException {
     if (editorWasOpen && type != FileActivity.Type.REMOVED) editorManager.openEditor(path, true);
   }
 
-  private void handleFileMove(FileActivity activity) throws CoreException {
+  private void handleFileMove(FileActivity activity)
```

Comment:
I still need to check whether moving the resource also preserves the encoding setting. If not, the encoding of the moved file has to be set, even if no content is passe (use encoding of old version of file in such cases).

Reply:
Move preservers file encoding.